File: HackerRank Solutions/Algorithms/Simple Array Sum.cpp
#include <bits/stdc++.h>
using namespace std;

vector<string> split_string(string);

/*
 * Complete the simpleArraySum function below.
 */
int main() {
    int n, sum = 0;
    cin >> n;
    vector<int> arr(n);
    for(int i=0; i<n; i++){
        cin >> arr[i];
    }
    for(int i=0; i<n; i++){
        sum += arr[i];
    }
    cout << sum;
    return 0;
}

File: HackerRank Solutions/Algorithms/Simple Array Sum.py
#!/bin/python3

import os
import sys

#
# Complete the simpleArraySum function below.
#
def simpleArraySum(ar):
    #
    # Write your code here.
    #
    sum = 0
    for i in range(len(ar)):
        sum = sum + ar[i]
    return sum
if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    ar_count = int(input())

    ar = list(map(int, input().rstrip().split()))

    result = simpleArraySum(ar)

    fptr.write(str(result) + '\n')

    fptr.close()
